proposals/all_proposals.md

```diff
@@ -0,0 +1,671 @@
+
+<lynX> This seems to be asking for an exchange standard, not a legislation. Standard document formats however exist, also means for upload — so if the industry hasn't established a standard practice in this field maybe that's because they have no economic incentive to do so? A legal policy could make it mandatory for publishing platforms to accept a uniform simple document upload format. Not clear to me which political problem this really addresses.
+
+# 2 
+# 28.06.2018 Input DSC Vienna1 - 18-06-28 
+
+First of all, the participants determined that they do not have a profound knowledge on the topic. The topic literally is very technical and interdisciplinary as well due to its interconnection and impacts to other political fields, but we nevertheless tried our best ;-). We ask for your understanding that we are contributing to the first block of questions democratising technologies only.
+One member stated that the fact that older citizens cannot use modern technologies due to physical reasons is to be considered as highly undemocratic (e.g. reading on a smartphone). More tax-generated public investments shall be undertaken in that field (research).
+<lynX> We could require that all smartphone apps must also be usable from any computer. Technically this is already possible, but hardly anyone has an Android emulator on their laptop.
+There was a consensus that the (legal) tax evasion (company HQ in other country, example amazon, apple etc.) is specifically an issue with tech companies.
+ <lynX> EC might indeed take care of that, but it is only a secondary problem. The main problem is how the advertising and online markets were allowed to converge onto just 5 gatekeeper companies (plus 2 in China).
+Democratic participation shall be ensured by democratically legitimated representatives (with both observation function and participation in decision making, maybe even the right for a suspensory veto if she/he sees a danger for the society - until a council of experts (professors etc.) decided on the question suspended) in high-level decision-making bodies in companies. The number of seats (e.g. in a supervisory board) could be dependent on the turnover or value of the company in all over Europe (in order to avoid workarounds by the establishment of subsidiaries). Currently in Austria, public authorities (not democratically legitimated) do have sort of an observation position in banks which we considered as not sufficient.
+ <lynX> Patchworky proposal. Is there any scientific sociologic backing to doing things this way? Liquid democracy at least has http://arxiv.org/abs/1503.07723 speaking for it. Also, representation doesn't work.
+We consider the harmonization of corporate taxation in Europe as an effective and even indispensable measure to ensure justice of balance in Europe.
```

Comment:
Probably true, but do we have scientific material on the subject? And does it belong to this pillar? - lynx